tests/queries/0_stateless/02715_exponential_smoothing_alpha.sql

```diff
@@ -0,0 +1,42 @@
+/* exponentialSmoothingAlpha tests */
+SELECT exponentialSmoothingAlpha()(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7 as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
```

Comment:
Instead of generating the same table data over and over, we could create them once at the beginning of the test (use [INSERT SELECT](https://clickhouse.com/docs/en/sql-reference/statements/insert-into#inserting-the-results-of-select) syntax) and then only reference these tables by the queries.

(also, numbers(100) will do the job just fine, the amount of data is too small to require multithreading)